plugins/usbdmx/AsyncPluginImpl.cpp

```diff
@@ -264,16 +264,9 @@ void AsyncPluginImpl::DeviceEvent(HotplugAgent::EventType event,
     // Sunlite plugin, if we make the f/w load async we'll need to let the
     // factory cancel the load.
 
-    // Unregister & delete the device in the main thread.
-    if (state->ola_device) {
-      Future<void> f;
-      m_plugin_adaptor->Execute(
-          NewSingleCallback(this, &AsyncPluginImpl::ShutdownDevice,
-                            state->ola_device, &f));
-      f.Get();
-      state->ola_device = NULL;
-    }
-    state->DeleteWidget();
+    // Unregister & delete the device and widget in the main thread.
+    m_plugin_adaptor->Execute(
+      NewSingleCallback(this, &AsyncPluginImpl::ShutdownDeviceState, state));
```

Comment:
Nit: Spacing tweak
```suggestion
    m_plugin_adaptor->Execute(
        NewSingleCallback(this, &AsyncPluginImpl::ShutdownDeviceState, state));
```

Reply:
Seems to me this file uses two space indent so I don't really understand why it's four spaces here, but since other parameter list continuations also use four space I suppose it's consistent at least :)

Reply:
Thanks. It's precisely because of that, it's parameter continuation, not another logical level of nesting.

This is essentially just wrapping to keep the line an acceptable length so we want to differentiate it from control flow.